dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/HibernateEventChangeLogStore.java

```diff
@@ -58,9 +58,9 @@ public class HibernateEventChangeLogStore {
   private static final String COLUMN_CHANGELOG_DATA_ELEMENT = "d.uid";
   private static final String COLUMN_CHANGELOG_FIELD = "ecl.eventField";
   private static final String ORDER_CHANGE_EXPRESSION =
-      "CONCAT(COALESCE(d.formName, ''), COALESCE(d.name, ''), COALESCE("
+      "CONCAT(COALESCE(LOWER(d.formName), ''), COALESCE(LOWER(d.name), ''), COALESCE(LOWER("
```

Comment:
> Additionally, I fixed a bug in event change log sorting. Our collation is case-sensitive, and I hadn’t taken that into account when I implemented the sorting for data elements.

Can you please elaborate? How is it supposed to behave vs how was it behaving?

Reply:
The DB collation is case sensitive, so the issue was that a list like `Clinic 1, clinic 2, Clinic 3`, would be sorted like this: `Clinic 1, Clinic 3, clinic 2`.

Reply:
Maybe worth discussing https://www.postgresql.org/docs/current/collation.html in the backend meeting. Is that always expected? if so why do we not configure the DB instead. Or is this something implementation admins should configure

Reply:
Yes, this I'll bring up next week.